src/preview/index.ts

```diff
@@ -1,36 +1,26 @@
 import fs from 'fs';
 import { compile } from 'handlebars';
-import templates from '../templates';
-import { getProposals } from '../helpers/snapshot';
-import styles from '../helpers/styles.json';
+import prepareSubscribe from '../templates/subscribe';
+import prepareSummary from '../templates/summary';
 
 export default async function preview(req, res) {
-  const template = templates[req.params.template];
-
-  if (!template) {
-    return res.sendStatus(404);
-  }
-
   const params = {
     to: req.query.to || 'fabien@bonustrack.co',
-    name: req.query.name || 'Fabien',
-    address: req.query.address || '0xeF8305E140ac520225DAf050e2f71d5fBcC543e7',
-    proposals: {},
-    styles
+    address: req.query.address || '0xeF8305E140ac520225DAf050e2f71d5fBcC543e7'
   };
+  let msg;
 
-  if (req.params.template === 'summary') {
-    params.proposals = await getProposals(params.address);
+  switch (req.params.template) {
+    case 'subscribe':
+      msg = await prepareSubscribe(params);
+      break;
+    case 'summary':
+      msg = await prepareSummary(params);
+      break;
+    default:
+      return res.sendStatus(404);
```

Comment:
Instead of this we could import all template directly and just do 
```
if (templates[req.params.template]) {
  await templates[req.params.template].prepare;
}
```
It would be less line of code and would be less setup to add new template

Reply:
All emails building logic are now called via the `prepare` method inside templates, inside of being called directly